kornia/core/image.py

```diff
@@ -0,0 +1,136 @@
+from enum import Enum
+from typing import List, Tuple, Union
+
+from kornia.color import bgr_to_grayscale, bgr_to_rgb, rgb_to_bgr, rgb_to_grayscale
+from kornia.core import Tensor
+from kornia.geometry.transform import hflip, vflip
+from kornia.utils import image_to_tensor, tensor_to_image
+
+
+class ImageColor(Enum):
+    GRAY = 0
+    RGB = 1
+    BGR = 2
+
+
+class Image(Tensor):
```

Comment:
Add documentation, that image is expected in (B)CHW format

Reply:
Definitely, one question is how do we define video frames. @lferraz